probe-rs/src/architecture/arm/sequences/stm32h7.rs

```diff
@@ -161,14 +163,21 @@ impl ArmDebugSequence for Stm32h7 {
         Ok(())
     }
 
-    fn debug_core_stop(&self, interface: &mut dyn ArmProbeInterface) -> Result<(), ArmError> {
-        // Power up the debug components through AP2, which is the defualt AP debug port.
-        let ap = MemoryAp::new(ApAddress {
+    fn debug_core_stop(
+        &self,
+        interface: &mut dyn ArmProbeInterface,
+        _core_ap: MemoryAp,
+        _core_type: CoreType,
+    ) -> Result<(), ArmError> {
+        // Power up the debug components through AP2, which is the default AP debug port.
+
+        let debug_ap = MemoryAp::new(ApAddress {
```

Comment:
Why not use the `core_ap` here?

Reply:
The previous implementation didn't use the AP for the core either, the AP 2 is the debug AP for STM32H7 devices.